src/main/java/ru/clevertec/userservice/user/controller/UserController.java

```diff
@@ -2,79 +2,73 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import ru.clevertec.userservice.user.model.dto.UserDto;
 import ru.clevertec.userservice.user.model.request.CreateUserRequest;
 import ru.clevertec.userservice.user.model.request.UpdateUserRequest;
-import ru.clevertec.userservice.user.model.response.ApiResponse;
 import ru.clevertec.userservice.user.service.UserService;
 
 import java.util.List;
 import java.util.UUID;
 
 @RestController
 @Slf4j
-@RequestMapping("/api/v1/users")
+@RequestMapping("/api/v1/admin/users")
 @RequiredArgsConstructor
 public class UserController {
     private final UserService userService;
 
     @PostMapping
-    public ApiResponse<UserDto> createUser(@RequestBody CreateUserRequest request) {
-
+    public ResponseEntity<UserDto> createUser(@RequestBody CreateUserRequest request) {
         log.info("###POST createUserRequest = {}", request);
-
         UserDto userDto = userService.saveUser(request);
         log.info("userDto = {}", userDto);
-        return new ApiResponse<>(userDto);
+        return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
     @GetMapping("/email/{email}")
-    public ApiResponse<UserDto> getUserByEmail(@PathVariable String email) {
+    public ResponseEntity<UserDto> getUserByEmail(@PathVariable String email) {
         log.info("###GET email = {}", email);
         UserDto userDto = userService.getUserByEmail(email);
         log.info("-------userDto = {}", userDto);
-        return new ApiResponse<>(userDto);
+        return new ResponseEntity<>(HttpStatus.FOUND);
     }
 
     @GetMapping
-    public ApiResponse<List<UserDto>> getAllUsers() {
-
+    public ResponseEntity<List<UserDto>> getAllUsers() {
         log.info("###GET getAllUsers");
-
-        return new ApiResponse<>(userService.getAllUsers());
+        userService.getAllUsers();
+        return new ResponseEntity<>(HttpStatus.FOUND);
     }
 
     @GetMapping("/{id}")
-    public ApiResponse<UserDto> getUserById(@PathVariable UUID id) {
-
+    public ResponseEntity<UserDto> getUserById(@PathVariable UUID id) {
         log.info("###GET fetched id = {}", id);
-
-        return new ApiResponse<>(userService.getUserById(id));
+        userService.getUserById(id);
+        return new ResponseEntity<>(HttpStatus.FOUND);
     }
 
     @PutMapping
-    public ApiResponse<UserDto> updateUser(@RequestBody UpdateUserRequest request) {
-
+    public ResponseEntity<UserDto> updateUser(@RequestBody UpdateUserRequest request, @RequestParam UUID roleId) {
```

Comment:
remove ResponseEntity wrapper if you are using OK status

Reply:
add role id as @PathVariable

Reply:
fixed